Symbiflow not working for Xilinx

Similar to the issue #722 I am seeing that synthesis is failing 

```
Project status:
    [S] bitstream:  bitstream -> /home/navaneeth/Projects/openmpw/CFU-Playground/soc/build/digilent_arty.proj_template/gateware/digilent_arty.bit
    [N] build_dir:  /home/navaneeth/Projects/openmpw/CFU-Playground/soc/build/digilent_arty.proj_template/gateware
    [S] eblif:  synth -> /home/navaneeth/Projects/openmpw/CFU-Playground/soc/build/digilent_arty.proj_template/gateware/digilent_arty.eblif
    [S] fasm:  fasm -> /home/navaneeth/Projects/openmpw/CFU-Playground/soc/build/digilent_arty.proj_template/gateware/digilent_arty.fasm
    [S] fasm_extra:  synth -> /home/navaneeth/Projects/openmpw/CFU-Playground/soc/build/digilent_arty.proj_template/gateware/digilent_arty_fasm_extra.fasm
    [S] io_place:  ioplace -> /home/navaneeth/Projects/openmpw/CFU-Playground/soc/build/digilent_arty.proj_template/gateware/digilent_arty.ioplace
    [S] net:  pack -> /home/navaneeth/Projects/openmpw/CFU-Playground/soc/build/digilent_arty.proj_template/gateware/digilent_arty.net
    [X] pcf:  MISSING
    [S] place:  place -> /home/navaneeth/Projects/openmpw/CFU-Playground/soc/build/digilent_arty.proj_template/gateware/digilent_arty.place
    [S] place_constraints:  place_constraints -> /home/navaneeth/Projects/openmpw/CFU-Playground/soc/build/digilent_arty.proj_template/gateware/digilent_arty.preplace
    [S] route:  route -> /home/navaneeth/Projects/openmpw/CFU-Playground/soc/build/digilent_arty.proj_template/gateware/digilent_arty.route
    [S] sdc:  synth -> /home/navaneeth/Projects/openmpw/CFU-Playground/soc/build/digilent_arty.proj_template/gateware/digilent_arty.sdc
    [N] sources:  ['/home/navaneeth/Projects/openmpw/CFU-Playground/proj/proj_template/cfu.v', '/home/navaneeth/Projects/openmpw/CFU-Playground/third_party/python/pythondata_cpu_vexriscv/pythondata_cpu_vexriscv/verilog/VexRiscv_FullCfu.v', '/home/navaneeth/Projects/openmpw/CFU-Playground/soc/build/digilent_arty.proj_template/gateware/digilent_arty.v']
    [N] xdc:  /home/navaneeth/Projects/openmpw/CFU-Playground/soc/build/digilent_arty.proj_template/gateware/digilent_arty.xdc

Executing module `synthesize`:
    [1/3] : Synthesizing sources: ['/home/navaneeth/Projects/openmpw/CFU-Playground/proj/proj_template/cfu.v', '/home/navaneeth/Projects/openmpw/CFU-Playground/third_party/python/pythondata_cpu_vexriscv/pythondata_cpu_vexriscv/verilog/VexRiscv_FullCfu.v', '/home/navaneeth/Projects/openmpw/CFU-Playground/soc/build/digilent_arty.proj_template/gateware/digilent_arty.v']...
[ERROR]: yosys non-zero return code.
stderr:
ERROR: TCL interpreter returned an error: Yosys command produced an error



make[5]: *** [/home/navaneeth/Projects/openmpw/CFU-Playground/soc/common_soc.mk:115: build/digilent_arty.proj_template/gateware/digilent_arty.bit] Error 1
make[5]: Leaving directory '/home/navaneeth/Projects/openmpw/CFU-Playground/soc'
make[4]: *** [../proj.mk:319: prog] Error 2
make[4]: Leaving directory '/home/navaneeth/Projects/openmpw/CFU-Playground/proj/proj_template'
```
In `digilent_arty_synth.log`

```
-- Running command `tcl /home/navaneeth/Projects/openmpw/CFU-Playground/env/symbiflow/xc7/install/share/f4pga/scripts/xc7/synth.tcl' --
[TCL: yosys -import] Command name collision: found pre-existing command `cd' -> skip.
[TCL: yosys -import] Command name collision: found pre-existing command `eval' -> skip.
[TCL: yosys -import] Command name collision: found pre-existing command `exec' -> skip.
[TCL: yosys -import] Command name collision: found pre-existing command `read' -> skip.
[TCL: yosys -import] Command name collision: found pre-existing command `trace' -> skip.
ERROR: Can't load module `./xdc': /home/navaneeth/Projects/openmpw/CFU-Playground/env/conda/envs/cfu-symbiflow/bin/../share/yosys/plugins/xdc.so: undefined symbol: _ZN5Yosys7stringfB5cxx11EPKcz
ERROR: TCL interpreter returned an error: Yosys command produced an error
```

Commands I have used are
```
make install-sf
make enter-sf
make prog TARGET=digilent_arty USE_SYMBIFLOW=1
```

--- a/Makefile
+++ b/Makefile
@@ -63,7 +63,8 @@ install-sf:
 	wget -qO- https://storage.googleapis.com/symbiflow-arch-defs/artifacts/prod/foss-fpga-tools/symbiflow-arch-defs/continuous/install/${F4PGA_TIMESTAMP}/symbiflow-arch-defs-install-xc7-${F4PGA_HASH}.tar.xz | tar -xJC $(SF_INSTALL)/xc7/install
 	wget -qO- https://storage.googleapis.com/symbiflow-arch-defs/artifacts/prod/foss-fpga-tools/symbiflow-arch-defs/continuous/install/${F4PGA_TIMESTAMP}/symbiflow-arch-defs-xc7a50t_test-${F4PGA_HASH}.tar.xz | tar -xJC $(SF_INSTALL)/xc7/install
 	wget -qO- https://storage.googleapis.com/symbiflow-arch-defs/artifacts/prod/foss-fpga-tools/symbiflow-arch-defs/continuous/install/${F4PGA_TIMESTAMP}/symbiflow-arch-defs-xc7a100t_test-${F4PGA_HASH}.tar.xz | tar -xJC $(SF_INSTALL)/xc7/install
-	wget -qO- https://storage.googleapis.com/symbiflow-arch-defs/artifacts/prod/foss-fpga-tools/symbiflow-arch-defs/continuous/install/${F4PGA_TIMESTAMP}/symbiflow-arch-defs-xc7a200t_test-${F4PGA_HASH}.tar.xz | tar -xJC $(SF_INSTALL)/xc7/install
+# Uncomment the following line to enable xc7a200t
+#	wget -qO- https://storage.googleapis.com/symbiflow-arch-defs/artifacts/prod/foss-fpga-tools/symbiflow-arch-defs/continuous/install/${F4PGA_TIMESTAMP}/symbiflow-arch-defs-xc7a200t_test-${F4PGA_HASH}.tar.xz | tar -xJC $(SF_INSTALL)/xc7/install
 	$(MAKE) USE_SYMBIFLOW=1 env
 	@echo
 	@echo "Done installing SymbiFlow.  To enter the environment, type 'make enter-sf', which creates a new subshell, and 'exit' when done."

--- a/conf/environment-symbiflow.yml
+++ b/conf/environment-symbiflow.yml
@@ -12,7 +12,8 @@ dependencies:
   - litex-hub::nextpnr-nexus
   - litex-hub::nextpnr-ecp5
   - litex-hub::nextpnr-ice40
-  - litex-hub::yosys
+# Temporarily pin Yosys until symbiflow-yosys-plugins catches up
+  - litex-hub::yosys=0.33_11_g31ee566ec=20230724_080446_py37
   - litex-hub::iceprog
   - litex-hub::prjxray-tools
   - litex-hub::prjxray-db

--- a/conf/environment.yml
+++ b/conf/environment.yml
@@ -12,7 +12,8 @@ dependencies:
   - litex-hub::nextpnr-ecp5
   - litex-hub::nextpnr-ice40
   - litex-hub::iceprog
-  - litex-hub::yosys
+# Temporarily pin Yosys until symbiflow-yosys-plugins catches up
+  - litex-hub::yosys=0.33_11_g31ee566ec=20230724_080446_py37
   - litex-hub::symbiflow-yosys-plugins
   - libevent
   - json-c

